### PR TITLE
fix redirects to home

### LINKED ISSUE #66: Make "/" redirect to "/home"
Currently, http://pclubuiet.herokuapp.com/ leads to an error as no view responds to this request. 
The home page is at http://pclubuiet.herokuapp.com/home

Just make the former redirect to the latter.

### PR DESCRIPTION
**Pull request by guptarohit**
This fixes #66 . :star2:

### CODE REVIEW

**Comment by sagar-kalra** [comment]
@guptarohit issue says to redirect it to /home, not to create a seperate url for that.

File: home/urls.py [hunk 1]
```diff
@@ -4,4 +4,5 @@
 urlpatterns = [
 url(r'^$', views.Home.as_view(), name = 'home'),
 url(r'^resources/', views.Resources.as_view(), name = 'resources')
+
```
Comment (sagar-kalra) [inline home/urls.py]: Remove this extra line.

Comment (gurpreetsingh00885) [inline home/urls.py]: unnecessary change

Comment (guptarohit) [reply home/urls.py]: ok, fixing this.

**Comment by gurpreetsingh00885** [comment]
Your commits need to be squashed into a single commit.

**Comment by guptarohit** [comment]
@gurpreetsingh00885 should I create another branch ? or hard reset previous commits?

**Comment by sagar-kalra** [comment]
https://www.devroom.io/2011/07/05/git-squash-your-latests-commits-into-one/
at last just force push to your branch
@guptarohit

**Comment by guptarohit** [comment]
@sagar-kalra @gurpreetsingh00885 can you please tell what just happened? I squashed my previous two commits and kept only the latest one. But instead of showing latest commit message, it's showing commit by another person along with my changes added to that.
Seems like I things got messed up! Is the anyway I can fix this?

**Comment by sagar-kalra** [comment]
Rebase and push again, this might help. @guptarohit

**Comment by divyam3897** [comment]
Fixed in https://github.com/pclubuiet/website/commit/ce643e1afa7c07424f020cb052211013f0c5728a Thanks
